scipy/linalg/flapack_gen.pyf.src

```diff
@@ -221,6 +221,22 @@ subroutine <prefix>getrs(n,nrhs,lu,piv,b,info,trans)
 
 end subroutine <prefix>getrs
 
+subroutine <prefix>getc2(n,a,ipiv,jpiv,info)
```

Comment:
I based this on the wrapper for [`?getrf`](http://www.netlib.org/lapack/explore-html/dd/d9a/group__double_g_ecomputational_ga0019443faea08275ca60a734d0593e60.html) without really understanding: Why are `m` and `n` included in the signature:
```<prefix>getrf(m,n,a,piv,info)```
when they will be removed from the list of required or optional arguments according to `intent(hide)`? Whatever the answer is, why does the same argument not apply to lda?

Reply:
Dgetrf is for any rectangular array. This is specific for square arrays

Reply:
Yes, but I'm asking why they are included in the subroutine signature when they are going to be hidden from the user?
Why don't we omit them from the subroutine signature like `lda`?

Reply:
Any intermediate variable (including the ones which are derived automatically) to be used needs to be declared in the subroutine signature. Then you can use their pointer `&m`, `&n` in the f2py callstatement. Same story with `lda`.